views/beers.hbs

```diff
@@ -0,0 +1,62 @@
+<!DOCTYPE html>
```

Comment:
If you use a layout-file, you don't need to add the HTML-structure to the beers-view.